fesvr/htif.h

```diff
@@ -63,7 +63,7 @@ class htif_t : public chunked_memif_t
   virtual void idle() {}
 
   const std::vector<std::string>& host_args() { return hargs; }
-  std::vector<std::string>& target_args() { return targs; }
+  const std::vector<std::string>& target_args() { return targs; }
```

Comment:
Does this still need to be `protected` instead of `private`?

Reply:
Yes, we'd still like a way for subclasses to read targs after htif populates it.